cfpb/cfgov-refresh
# Mega Menu and secondary nav: update hover and select colors

## Changes

- Make menu links hover green and selected hover black.
- Make secondary nav link selected hover black.
- Remove href from selected secondary nav link.

## Testing

1. Pull branch and build.
2. Visit http://localhost:8000/policy-compliance/rulemaking/rules-under-development/ and explore the secondary nav link hover states. 
3. Manually add `o-mega-menu_content-2-link__current` to one of the menu links to see the selected state there since the current menu item is not currently highlighted.
 
## Screenshots

![secnav](https://user-images.githubusercontent.com/704760/76565539-2b67c080-6481-11ea-8ccf-a95738bec425.gif)
